packages/thrift-integration/src/client/connections/HttpConnection.spec.ts

```diff
@@ -601,7 +601,7 @@ describe('HttpConnection', () => {
         })
 
         after(async () => {
-            return new Promise((resolve, reject) => {
+            return new Promise<void>((resolve, reject) => {
```

Comment:
Addressing breaking change: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-1.html#resolves-parameters-are-no-longer-optional-in-promises